src/domain/QuizAnswer.ts

```diff
@@ -0,0 +1,7 @@
+export interface QuizAnswer {
+  roleId: number
+  answers: {
```

Comment:
Going beyong: You could create another interface, in this same file, called Answer. And then you have answers field here, like this: Answer[]. What do you think?